astropy/modeling/bounding_box.py

```diff
@@ -1418,7 +1418,8 @@ def validate(cls, model, bounding_box: dict, selector_args=None, create_selector
             bounding_box = bounding_box.bounding_boxes
 
         if selector_args is None:
-            raise ValueError("Selector arguments must be provided (can be passed as part of bounding_box argument)!")
+            raise ValueError("Selector arguments must be provided "
+                             "(can be passed as part of bounding_box argument)!")
```

Comment:
Maybe out of scope but I can't help but notice in this diff... is the exclamation mark really necessary? 
```suggestion
                             "(can be passed as part of bounding_box argument)")
```

Reply:
Why not!. See changes in 953ebe4.